public/keycloak/keycloak.ts

```diff
@@ -3,8 +3,14 @@ import { KCUserProfile } from '~/public/keycloak/KCUserProfile'
 import ConfigHelper from '~/util/config-helper'
 import { SessionStorageKeys } from '~/util/constants'
 import { Store } from 'pinia'
-import { getModule } from 'vuex-module-decorators'
-import AuthModule from '../store/modules/auth'
+import { createPinia } from 'pinia';
+import { createApp } from 'vue'
+import App from '~/app.vue'
+const pinia = createPinia()
```

Comment:
I guess the code doesn't care but it's nicer to see all imports at the top and then have some visual separation (a blank line) to the code. What do you think?

Reply:
I've added some visual separation and comments to clarify, however, the reason this is structured this was is that const pinia = createPinia()
const app = createApp(App)
app.use(pinia)
need to be called BEFORE I can import { AuthModule } from '~/store/auth' for pinia to function correctly.

Reply:
Ah, OK. Then the comments will be very helpful to the next developer so they don't try to rearrange this :)

Reply:
was able to segregate imports from the code